src/components/Legend.tsx

```diff
@@ -209,39 +213,56 @@ export function Legend({
   );
 }
 
-function getLineShape(config: { index: number; color: string }) {
+function getLineShape(config: {
+  index: number;
+  color: string;
+  style?: CSSProperties;
+}) {
   const x = 0;
-  const y = (config.index + 1) * 16 - x - 7;
+  //hard coding values should be fixed
+  const y = (config.index + 1) * 16 - x - 5;
 
-  return <rect x={x} y={y} width="20" height="3" fill={config.color} />;
```

Comment:
i found that line in legend was a hard codded rect so i used a line instead (we should fix y position when label fontSize changes)